runtime/doc/eval.txt

```diff
@@ -7470,6 +7486,7 @@ localmap		Compiled with local mappings and abbr. |:map-local|
 mac			Macintosh version of Vim.
 macunix			Macintosh version of Vim, using Unix files (OS-X).
 menu			Compiled with support for |:menu|.
+msgpane			Compiled with support for 'messagepane'.
```

Comment:
I copied a pattern.  I don't know if it's needed in the docs.  I wanted to have `has('msgpane')` so plugin developers could have some expectation of how messages are presented.